openeo/extra/job_management.py

```diff
@@ -0,0 +1,379 @@
+import collections
+import contextlib
+import datetime
+import json
+import logging
+import os
+import time
+from pathlib import Path
+from typing import Callable, Union, Dict, Optional
+
+import geopandas as gpd
+import pandas as pd
+import requests
+from openeo import Connection, BatchJob
+from openeo.rest import OpenEoApiError
+from openeo.util import deep_get
+
+from openeo.rest.connection import cached_connection
+
+
+_log = logging.getLogger(__name__)
+
+
+def run_jobs(
```

Comment:
I believe we don't actually need this method? This was an older version, before we had MultibackendJobManager

Reply:
Has been removed